packages/bpk-docs/src/pages/ThemingPage/ThemingPage.js

```diff
@@ -85,18 +80,16 @@ const webBlurb = [
   <Paragraph>The following web components are themeable:</Paragraph>,
   <BpkList>
     <BpkListItem>
-      <BpkRouterLink to={ACCORDIONS}>Accordion</BpkRouterLink>
+      <BpkRouterLink to={ACCORDION}>Accordion</BpkRouterLink>
     </BpkListItem>
     <BpkListItem>
-      <BpkRouterLink to={BARCHARTS}>Bar chart</BpkRouterLink>
+      <BpkRouterLink to={BARCHART}>Bar chart</BpkRouterLink>
     </BpkListItem>
     <BpkListItem>
-      <BpkRouterLink to={`${TYPOGRAPHY}#blockquotes`}>
-        Blockquotes
-      </BpkRouterLink>
+      <BpkRouterLink to={BLOCKQUOTE}>Blockquotes</BpkRouterLink>
     </BpkListItem>
     <BpkListItem>
-      <BpkRouterLink to={BUTTONS}>Button</BpkRouterLink>
+      <BpkRouterLink to={BUTTON}>Button</BpkRouterLink>
```

Comment:
Any reason we're not linking to the web tab (platform=web) here too? Same question for all bellow that have a native version.

Reply:
Good shout, at the moment the link on both native and web goes to native so I'll update that to improve it